### PR TITLE
Android Spaces SDK example

### PR DESCRIPTION
**Pull request by nbirkenshaw-mux**
This creates the example as used in the Android Getting Started guide [here](https://www.notion.so/Real-Time-Video-Getting-Started-Guide-for-Android-b21776d9a32d492ca4924176d4342b1d)

It requires manually replacing the JWT by the end user.

### CODE REVIEW

File: android-spaces-real-time-app/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Mux Spaces SDK Example
+
+This is an Android app demonstrating using the Spaces SDK to add real-time video and audio to your app. It is the same code as developed in the Getting Started guide [here](https://www.notion.so/Real-Time-Video-Getting-Started-Guide-for-Android-b21776d9a32d492ca4924176d4342b1d)
```
Comment (philcluff) [inline android-spaces-real-time-app/README.md:3]: Let's remember to update this to point at the published version when it's ready.

File: android-spaces-real-time-app/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Mux Spaces SDK Example
```
Comment (philcluff) [inline android-spaces-real-time-app/README.md:1]: Nit: Mention Android in the title here.

File: android-spaces-real-time-app/app/src/main/java/com/mux/realtimevideoexample/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+public class MainActivity extends AppCompatActivity {
+    private static final String[] PERMISSIONS = new String[] {
+        Manifest.permission.CAMERA,
+        Manifest.permission.RECORD_AUDIO
+    };
+
+    private final Space.Listener spaceListener = new Space.Listener() {
+        @Override
+        public void onJoined(Space space, LocalParticipant localParticipant) {
+            Toast.makeText(MainActivity.this, "Joined space "+space.getId()+" as "+localParticipant.getId(), Toast.LENGTH_SHORT).show();
```
Comment (philcluff) [inline android-spaces-real-time-app/app/src/main/java/com/mux/realtimevideoexample/MainActivity.java:34]: Nit: Let's make all these `LENGTH_LONG` (which in my opinion is still too short - I always miss these)

**Review by philcluff — COMMENTED** [review]
Looks generally fine, some minor nits. but needs cross linking from the README.md, and needs its own README.md in this firectory.

**Review by andrewjl-mux — APPROVED** [review]

**Pull request merged**